### ISSUE TITLE
Refactor workflows templates and move to invenio_workflows_ui

### ISSUE THREAD

**Issue by david-caro** [inspirehep/inspire-next#2080]

**Comment by jmartinm** [comment]
I don't think templates should be moved. https://github.com/inveniosoftware-contrib/invenio-workflows-ui acts like the rest of `invenio-foo-ui` packages which is basically handling the main views and maybe permissions (with sample templates)

Templates are in the overlay and that eases development and finding where things are in the codebase.

**Comment by jacquerie** [comment]
> Templates are in the overlay and that eases development and finding where things are in the codebase.

We could have both by continuing the `submodules` thing you started for `inspirehep-search-js`: code is neatly separated in repos, but you can grep easily the codebase as long as you checkout the submodules locally.

(If you remember there was a speed talk at dotScale about this approach.)

**Comment by jmartinm** [comment]
> We could have both by continuing the submodules thing you started for inspirehep-search-js: code is neatly separated in repos, but you can grep easily the codebase as long as you checkout the submodules locally.

Right, I still think the disadvantage of having things spread out in `inveniosoftware`, `inveniosoftware-contrib` and `inspirehep` outweighs the benefits of testability  and neat separation.

I am kind of ok with separating core packages that need to be reused and do not change often but when it comes to data model specific (such as templates) it is way more convenient to have in `inspire-next`.

**Comment by jmartinm** [comment]
@david-caro I have amended title/description to clarify more the task to be done.

**Comment by david-caro** [comment]
👍
